common/src/main/java/io/pravega/common/util/IdGenerator.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.util;
+
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.LongUnaryOperator;
+
+/**
+ * IdGenerator is used to generated unique ids which are guaranteed to be unique per process. The id
```

Comment:
"unique ids which are guaranteed to be unique per process" -> "ids which are guaranteed to be unique per process"